drivers/usb/udc/udc_numaker.c

```diff
@@ -1371,8 +1371,8 @@ static void numaker_udbd_isr(const struct device *dev)
 			 */
 			if (ep == USB_EP_GET_ADDR(0, USB_EP_DIR_OUT)) {
 				struct numaker_usbd_ep *ep_ctrlout = priv->ep_pool + 0;
-				USBD_EP_T *ep_ctrlout_base =
-					numaker_usbd_ep_base(dev, ep_ctrlout->ep_hw_idx);
+				USBD_EP_T *ep_ctrlout_base = numaker_usbd_ep_base(
+					dev, ep_ctrlout->ep_hw_idx);
```

Comment:
This kind of reformatting does not make it better. numaker_udbd_isr() function is quite large, I would split it in a few smaller and more specific routines, plug/unplug, wakeup, setup and transaction/transfer handling. Better code has fewer indents.